r2i/ncsdk/engine.h

```diff
@@ -12,13 +12,13 @@
 #ifndef R2I_NCSDK_ENGINE_H
 #define R2I_NCSDK_ENGINE_H
 
-#include <r2i/iengine.h>
+#include <r2i/engine.h>
 #include <r2i/ncsdk/model.h>
 
 namespace r2i {
 namespace ncsdk {
 
-class Engine : public IEngine {
+class Engine : public r2i::Engine {
```

Comment:
Since now these frameworks are now inheriting from the engine class, I suspect that there's q bunch of methods that may be removed.